packages/mobile/src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -74,12 +74,9 @@ const ExploreStackScreen = createStackScreen<ExploreStackParamList>(Stack => (
       name='TrendingUnderground'
       component={TrendingUndergroundScreen}
     />
-    <Stack.Screen
-      name='UnderTheRadar'
-      component={() => (
-        <SmartCollectionScreen smartCollection={UNDER_THE_RADAR} />
-      )}
-    />
+    <Stack.Screen name='UnderTheRadar'>
+      {() => <SmartCollectionScreen smartCollection={UNDER_THE_RADAR} />}
```

Comment:
yeah good catch here, wondering if we just create an under-the-radar screen maybe?

Reply:
Probably a good idea, we will need to create them for each smart collection though